src/tools/kdb/meta-get.c

```diff
@@ -29,7 +32,59 @@ void addMetaGetSpec (KeySet * spec)
 	ADD_BASIC_OPTIONS (spec, COMMAND_SPEC_KEY (COMMAND_NAME))
 }
 
-int execCppMetaGet (int argc, char ** argv)
+int execMetaGet (KeySet * options, Key * errorKey)
 {
-	return cpp_main (argc, argv);
+	int ret = 0;
+	GET_BASIC_OPTIONS
+
+	Key * toLookUp = getKeyFromOptions (GET_OPTION (options, "keyname"), errorKey, verbose);
+	if (toLookUp == NULL)
+	{
+		RETURN (2)
+	}
+
+	const char * metaName = GET_OPTION (options, "metaname");
+
+	KeySet * conf = ksNew (0, KS_END);
+	KDB * handle = kdbOpen (NULL, errorKey);
+
+	Key * cascadingParent = keyNew ("/", KEY_END);
```

Comment:
Why this parent? This probably loads far more data than necessary.

It's also inconsistent with all the other `kdb meta` commands and especially `kdb meta show`, which I'd expect to print the same value for a given meta name, so both should be able to use the same parent.

Reply:
Good question, sometimes past hannes does weird stuff, fixed it :)